apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -47,10 +47,6 @@ const styles = {
   }
 };
 
-/**
- * Equivalent of visualizationColumn.html.ejs. Initially only supporting
- * portions used by App Lab
- */
```

Comment:
Removing this comment because the referenced file doesn't exist anymore and the name includes AppLab, so it's safe to assume the component only supports AppLab 😄